resources/assets/js/helpers.js

```diff
@@ -77,3 +77,24 @@ export function relativeDateTime(value) {
 export function diffInSeconds(otherMoment) {
     return moment().diff(otherMoment, 'seconds');
 }
+
+export function positionToGridAreaNotation(position) {
+
+    function indexInAlphabet(character) {
```

Comment:
move this function outside of `positionToGridAreaNotation`. If you don't export it it will be private.